pkg/spec/postgresql_test.go

```diff
@@ -43,14 +43,29 @@ var clusterNames = []struct {
 	{"acid-test", "acid", "test", nil},
```

Comment:
seems like "foobar" is not covered but should work as this is team "foo" and cluster "bar" in ec2 spilo way